<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>simuladorcomuClass</class>
 <widget class="QWidget" name="simuladorcomuClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>simuladorcomu</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QFormLayout" name="form_botonera">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_header1">
       <property name="text">
        <string>header1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineedit_header1"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lbl_header2">
       <property name="text">
        <string>header2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineedit_header2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbl_contador">
       <property name="text">
        <string>contador</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineedit_contador"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lbl_ref_est">
       <property name="text">
        <string>ref_est</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="checkbox_ref_est"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lbl_pant_datos">
       <property name="text">
        <string>pant_datos</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="checkbox_pant_datos"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lbl_pant_barrido">
       <property name="text">
        <string>pant_barrido</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="checkbox_pant_barrido"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lbl_pant_normal">
       <property name="text">
        <string>pant_normal</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lbl_vent_opt_lav">
       <property name="text">
        <string>vent_opt_lav</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="checkbox_vent_opt_lav"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lbl_vent_opt_llimp">
       <property name="text">
        <string>vent_opt_limp</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="checkbox_vent_opt_limp"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lbl_pa_ir">
       <property name="text">
        <string>pa_ir</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QCheckBox" name="checkbox_pa_ir"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lbl_cam_ir">
       <property name="text">
        <string>cam_ir</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="checkbox_cam_ir"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="lbl_cam_diu">
       <property name="text">
        <string>cam_diu</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QCheckBox" name="checkbox_cam_diu"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="lbl_zoom_amp">
       <property name="text">
        <string>zoom_amp</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="checkbox_zoom_amp"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="lbl_zoom_est">
       <property name="text">
        <string>zoom_est</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QCheckBox" name="checkbox_zoom_est"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="lbl_foco_cerca">
       <property name="text">
        <string>foco_cerca</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QCheckBox" name="checkbox_foco_cerca"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="lbl_foco_lejos">
       <property name="text">
        <string>foco_lejos</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QCheckBox" name="checkbox_foco_lejos"/>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="lbl_video_cuant">
       <property name="text">
        <string>video_cuant</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="lbl_video_dif">
       <property name="text">
        <string>video_dif</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="lbl_video_nor">
       <property name="text">
        <string>video_nor</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="lbl_eng_cuant">
       <property name="text">
        <string>eng_cuant</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="lbl_eng_dif">
       <property name="text">
        <string>eng_dif</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="QLabel" name="lbl_mod_fij">
       <property name="text">
        <string>mod_fij</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="lbl_mod_esc">
       <property name="text">
        <string>mod_esc</string>
       </property>
      </widget>
     </item>
     <item row="23" column="0">
      <widget class="QLabel" name="lbl_mod_est">
       <property name="text">
        <string>mod_est</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QCheckBox" name="checkbox_video_cuant"/>
     </item>
     <item row="17" column="1">
      <widget class="QCheckBox" name="checkbox_video_dif"/>
     </item>
     <item row="18" column="1">
      <widget class="QCheckBox" name="checkbox_video_nor"/>
     </item>
     <item row="19" column="1">
      <widget class="QCheckBox" name="checkbox_eng_cuant"/>
     </item>
     <item row="20" column="1">
      <widget class="QCheckBox" name="checkbox_eng_dif"/>
     </item>
     <item row="21" column="1">
      <widget class="QCheckBox" name="checkbox_mod_fij"/>
     </item>
     <item row="22" column="1">
      <widget class="QCheckBox" name="checkbox_mod_esc"/>
     </item>
     <item row="23" column="1">
      <widget class="QCheckBox" name="checkbox_mod_est"/>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="checkbox_pant_normal">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="from_pupitre">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_header1_pupitre">
       <property name="text">
        <string>header1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineedit_header1_pupitre"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
